src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs

```diff
@@ -231,7 +231,7 @@ private void BuildWalletConnectConfig()
 
     private IEnumerator FetchSupportedWallets()
     {
-        using (UnityWebRequest webRequest = UnityWebRequest.Get("https://registry.walletconnect.com/data/wallets.json"))
+        using (UnityWebRequest webRequest = UnityWebRequest.Get("https://explorer-api.walletconnect.com/v3/wallets?projectId=f4bff60eb260841f46b1c77588cd8acb"))
```

Comment:
Can this be replaced with `$"https://explorer-api.walletconnect.com/v3/wallets?projectId={projectId}"` makes it reference the serialized project Id

Reply:
can do

Reply:
Done